src/monobase/activate.sh

```diff
@@ -113,3 +113,5 @@ export LIBRARY_PATH="$CUDA_PATH/lib64/stubs"
 
 LD_CACHE_PATH="$MONOBASE_PATH/ld.so.cache.d/cuda$R8_CUDA_VERSION-cudnn$R8_CUDNN_VERSION-python$R8_PYTHON_VERSION"
 cp -f "$LD_CACHE_PATH" /etc/ld.so.cache
+
+export R8_ATTRIBUTES_FILES="${COG_VENV}/.done ${CUDA_PATH}/.done ${CUDNN_PATH}/.done ${VIRTUAL_ENV}/.done"
```

Comment:
Should we use `:` as separate to be consistent with `*PATH` env vars?

Reply:
I guess I didn't reach for `:` because I think of those as being ordered with first hit wins 🤷🏻‍♂️ I don't feel too strongly about it though.

Reply:
I guess I didn't reach for `:` because I think of those as being ordered with first hit wins 🤷🏻‍♂️ I don't feel too strongly about it though.